cmd/cosign/cli/generate/generate_key_pair.go

```diff
@@ -86,9 +89,9 @@ func GenerateKeyPairCmd(ctx context.Context, kmsVal string, args []string) error
 		return err
 	}
 
-	if fileExists("cosign.key") {
+	if fileExists(name + ".key") {
```

Comment:
maybe we can create a local variable that holds this whole name, then we can use it everywhere we need, WDYT?